periodic_job_test.go

```diff
@@ -42,7 +42,7 @@ func TestPeriodicJobBundle(t *testing.T) {
 			nil,
 		)
 
-		return newPeriodicJobBundle(newTestConfig(t, nil), periodicJobEnqueuer), &testBundle{}
```

Comment:
Made `schema` the second argument to `newTestConfig` instead, but honestly could see it with or without. It does save one LOC for test case where schemas are used (which is a lot of them), but it tends to be empty for test cases using the `runNewTestClient` helper, which injects its own schema.